.vib/moodle/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/moodle"
         },
-        "runtime_parameters": "Im1vb2RsZVBhc3N3b3JkIjogImJpdG5hbWkxIgoibW9vZGxlVXNlcm5hbWUiOiAidXNlciIKInNlcnZpY2UiOgogICJwb3J0IjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "bW9vZGxlVXNlcm5hbWU6IHRlc3RfdXNlcgptb29kbGVQYXNzd29yZDogQ29tcGxpY2F0ZWRQYXNzd29yZDEyMyE0CnNlcnZpY2U6CiAgdHlwZTogTG9hZEJhbGFuY2VyCiAgcG9ydHM6CiAgICBodHRwczogNDQ0Cm1hcmlhZGI6CiAgYXV0aDoKICAgIGRhdGFiYXNlOiB0ZXN0X21vb2RsZV9kYXRhYmFzZQogICAgdXNlcm5hbWU6IHRlc3RfbW9vZGxlX3VzZXIKcG9kU2VjdXJpdHlDb250ZXh0OgogIGZzR3JvdXA6IDEwMDIKY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogIHJ1bkFzVXNlcjogMTAwMgpjb250YWluZXJQb3J0czoKICBodHRwOiA4MDgxCiAgaHR0cHM6IDg0NDQ=",
```

Comment:
The PR looks good! 👍 One last comment, you missed adding the parameter `service.port.http` here. Even though we are not using it in Goss, the port is implicitly tested by Cypress once it connects.

Reply:
I've changed this now